### PR TITLE
Staging and Alfajores switch values

### PR DESCRIPTION
**Pull request by alvarof2**
### Description

_A few sentences describing the overall effects and goals of the pull request's commits.
What is the current behavior, and what is the updated/expected behavior with this PR?_

### Other changes

_Describe any minor or "drive-by" changes here._

### Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

### Related issues

- Fixes #[issue number here]

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### Documentation

_The set of community facing docs that have been added/modified because of this change_

### CODE REVIEW

File: docs/kubernetes-deployment/signer/values/mainnet/values-signer0-mainnet.yaml [hunk 1]
```diff
@@ -32,7 +35,7 @@ env:
     format: stackdriver
     level: trace
   odis:
-    odisSignerTimeout: "6000"
+    odisSignerTimeout: "10000"
     shouldMockAccountService: "false"
     shouldMockRequestService: "false"
     testQuotaBypassPercentage: "50"
```
Comment (soloseng) [inline docs/kubernetes-deployment/signer/values/mainnet/values-signer0-mainnet.yaml:41]: should be 0

File: docs/kubernetes-deployment/signer/values/mainnet/values-signer1-mainnet.yaml [hunk 2]
```diff
@@ -32,7 +35,7 @@ env:
     format: stackdriver
     level: trace
   odis:
-    odisSignerTimeout: "6000"
+    odisSignerTimeout: "10000"
     shouldMockAccountService: "false"
     shouldMockRequestService: "false"
     testQuotaBypassPercentage: "50"
```
Comment (soloseng) [inline docs/kubernetes-deployment/signer/values/mainnet/values-signer1-mainnet.yaml:41]: ditto

File: docs/kubernetes-deployment/signer/values/alfajores/values-signer0-alfajores.yaml [hunk 3]
```diff
@@ -33,7 +47,7 @@ env:
     level: trace
   odis:
     mockDek: 0x034846bc781cacdafc66f3a77aa9fc3c56a9dadcd683c72be3c446fee8da041070
-    odisSignerTimeout: "6000"
+    odisSignerTimeout: "10000"
     shouldMockAccountService: "false"
     shouldMockRequestService: "false"
     testQuotaBypassPercentage: "50"
```
Comment (soloseng) [inline docs/kubernetes-deployment/signer/values/alfajores/values-signer0-alfajores.yaml:53]: should be 0

File: docs/kubernetes-deployment/signer/values/alfajores/values-signer1-alfajores.yaml [hunk 4]
```diff
@@ -33,7 +47,7 @@ env:
     level: trace
   odis:
     mockDek: 0x034846bc781cacdafc66f3a77aa9fc3c56a9dadcd683c72be3c446fee8da041070
-    odisSignerTimeout: "6000"
+    odisSignerTimeout: "10000"
     shouldMockAccountService: "false"
     shouldMockRequestService: "false"
     testQuotaBypassPercentage: "50"
```
Comment (soloseng) [inline docs/kubernetes-deployment/signer/values/alfajores/values-signer1-alfajores.yaml:53]: ditto

File: docs/kubernetes-deployment/signer/values/alfajores/values-signer2-alfajores.yaml [hunk 5]
```diff
@@ -33,7 +47,7 @@ env:
     level: trace
   odis:
     mockDek: 0x034846bc781cacdafc66f3a77aa9fc3c56a9dadcd683c72be3c446fee8da041070
-    odisSignerTimeout: "6000"
+    odisSignerTimeout: "10000"
     shouldMockAccountService: "false"
     shouldMockRequestService: "false"
     testQuotaBypassPercentage: "50"
```
Comment (soloseng) [inline docs/kubernetes-deployment/signer/values/alfajores/values-signer2-alfajores.yaml:53]: ditto

File: docs/kubernetes-deployment/signer/values/alfajores/values-signer0-alfajores.yaml [hunk 6]
```diff
@@ -11,6 +22,7 @@ env:
   blockchain:
     blockchainApiKeyExistingSecret: odis-signer0-forno-key
     blockchainProvider: https://alfajores-forno.celo-testnet.org
+    fullNodeTimeoutMs: 10000
```
Comment (soloseng) [inline docs/kubernetes-deployment/signer/values/alfajores/values-signer0-alfajores.yaml:25]: I dont think this is part of the blockchain config.

File: docs/kubernetes-deployment/signer/values/alfajores/values-signer1-alfajores.yaml [hunk 7]
```diff
@@ -11,6 +22,7 @@ env:
   blockchain:
     blockchainApiKeyExistingSecret: odis-signer1-forno-key
     blockchainProvider: https://alfajores-forno.celo-testnet.org
+    fullNodeTimeoutMs: 10000
```
Comment (soloseng) [inline docs/kubernetes-deployment/signer/values/alfajores/values-signer1-alfajores.yaml:25]: ditto

File: docs/kubernetes-deployment/signer/values/alfajores/values-signer2-alfajores.yaml [hunk 8]
```diff
@@ -11,6 +22,7 @@ env:
   blockchain:
     blockchainApiKeyExistingSecret: odis-signer2-forno-key
     blockchainProvider: https://alfajores-forno.celo-testnet.org
+    fullNodeTimeoutMs: 10000
```
Comment (soloseng) [inline docs/kubernetes-deployment/signer/values/alfajores/values-signer2-alfajores.yaml:25]: ditto

File: docs/kubernetes-deployment/signer/values/staging/values-signer0-staging.yaml [hunk 9]
```diff
@@ -11,6 +11,7 @@ env:
   blockchain:
     blockchainApiKeyExistingSecret: odis-signer-forno-key
     blockchainProvider: https://alfajores-forno.celo-testnet.org
+    fullNodeTimeoutMs: 10000
```
Comment (soloseng) [inline docs/kubernetes-deployment/signer/values/staging/values-signer0-staging.yaml:14]: ditto

File: docs/kubernetes-deployment/signer/values/staging/values-signer1-staging.yaml [hunk 10]
```diff
@@ -11,6 +11,7 @@ env:
   blockchain:
     blockchainApiKeyExistingSecret: odis-signer-forno-key
     blockchainProvider: https://alfajores-forno.celo-testnet.org
+    fullNodeTimeoutMs: 10000
```
Comment (soloseng) [inline docs/kubernetes-deployment/signer/values/staging/values-signer1-staging.yaml:14]: ditto

File: docs/kubernetes-deployment/signer/values/mainnet/values-signer0-mainnet.yaml (hunk 1 above)
Comment (soloseng) [reply docs/kubernetes-deployment/signer/values/mainnet/values-signer0-mainnet.yaml:41]: Sorry, this comment was meant for `testQuotaPercentage:"50"`. 

`testQuotaPercentage:"50"` can actually be removed from the chart.

**Comment by alvarof2** [comment]
Set `testQuotaBypassPercentage` to 0. Pending for chart upgrade:
- Remove `testQuotaBypassPercentage` value
- Vars should follow the layout [here](https://github.com/celo-org/social-connect/blob/0aca8b2148a7b23efc79e0ab51861275916feb07/apps/signer/src/config.ts#L167)

File: docs/kubernetes-deployment/signer/README.md [hunk 11]
```diff
@@ -29,23 +29,31 @@ ODIS signer is deployed in the following clusters:
 
 - Staging: cluster `integration-tests`
   - Signer0 in namespace `odis-signer0-staging` with this [`values-signer0-staging.yaml` file](./values/staging/values-signer0-staging.yaml).
-    - URL: `http://odis-signer0-staging.odis-signer0-staging:3000`. :warning: This is a URL internal to the cluster (i.e. not accessible from outside)!!
+    - URL internal: `http://odis-signer0-staging.odis-signer0-staging:3000`. :warning: This is a URL internal to the cluster (i.e. not accessible from outside)!!
```
Comment (soloseng) [inline docs/kubernetes-deployment/signer/README.md:32]: Is there any reason to keep both URL now, given that the external URL is the one that users and developperes will reference?

File: docs/kubernetes-deployment/combiner/values/values-mainnet.yaml [hunk 12]
```diff
@@ -62,7 +62,7 @@ env:
...
     # -- Env. Var DOMAIN_ODIS_SERVICES_SIGNERS
-    domainOdisServicesSigners: '[{"url": "https://odis.vladiatorlabs.io"},{"url": "https://mainnet-pgpnp-brazilsouth.azurefd.net"},{"url": "https://phone.chainlayerattestations.com"},{"url": "https://pnprivacy.wotrust.us"},{"url": "https://pgpnp.census.works"},{"url": "https://odis.keyko.rocks"},{"url": "https://odis.celo.spruceid.xyz"}]'
+    domainOdisServicesSigners: '[{"url": "https://odis.vladiatorlabs.io"},{"url": "http://odis-signer0-mainnet.odis-signer0-mainnet:3000"},{"url": "https://phone.chainlayerattestations.com"},{"url": "https://pnprivacy.wotrust.us"},{"url": "https://pgpnp.census.works"},{"url": "https://odis.keyko.rocks"},{"url": "https://odis.celo.spruceid.xyz"}]'
```
Comment (soloseng) [inline docs/kubernetes-deployment/combiner/values/values-mainnet.yaml:65]: For consistency, should the cLabs Signer URL be updated to point to the external URL ?

Same for staging and alfajores.

**Review by soloseng — APPROVED** [review]
Looks great! 

Left a few comments that might require updating the deployed combiner yaml, but otherwise 🚀

File: docs/kubernetes-deployment/signer/README.md (hunk 11 above)
Comment (alvarof2) [reply docs/kubernetes-deployment/signer/README.md:32]: I would say yes. From the point of view of the combiner:
- The internal URL is a direct connection (Combiner -> Signer).
- The external URL means going out of the cluster -> Outer internet -> cluster ingress (that is an NGIX) -> Signer.
So using the internal URL for the combiner is better/faster/less network hops.

It's true that users and developers cannot use the internal URL, but our combiner can take advantage of this internal URL :)

File: docs/kubernetes-deployment/combiner/values/values-mainnet.yaml (hunk 12 above)
Comment (alvarof2) [reply docs/kubernetes-deployment/combiner/values/values-mainnet.yaml:65]: Ditto as the previous comment :)

File: docs/kubernetes-deployment/signer/README.md (hunk 11 above)
Comment (soloseng) [reply docs/kubernetes-deployment/signer/README.md:32]: Thanks for the input!

File: docs/kubernetes-deployment/signer/values/alfajores/values-signer2-alfajores.yaml [hunk 13]
```diff
@@ -33,38 +46,57 @@ env:
     level: trace
   odis:
     mockDek: 0x034846bc781cacdafc66f3a77aa9fc3c56a9dadcd683c72be3c446fee8da041070
-    odisSignerTimeout: "6000"
+    odisSignerTimeout: "10000"
```
[the remark this answers is not in the document]
Comment (alvarof2) [reply docs/kubernetes-deployment/signer/values/alfajores/values-signer2-alfajores.yaml:49]: There might be better solutions, but Helm has a bad habit of [converting numbers to scientific notation](https://github.com/helm/helm/issues/3001) which I wasn't sure the NodeJS process could manage well.

It's true it only happens for big numbers though.

[the remark this answers is not in the document]
Comment (soloseng) [reply docs/kubernetes-deployment/signer/values/alfajores/values-signer2-alfajores.yaml:49]: Fine with me if it's something that was done by Helm. 
I was just wondering if there was an difference between how `odisSignerTimeout` & `fullNodeTimeout` would be processed.

**Review by alecps — APPROVED** [review]

**Pull request merged**